src/components/AppSidebarTab/AppSidebarTab.vue

```diff
@@ -86,8 +86,7 @@ export default {
 .app-sidebar__tab {
 	display: none;
 	padding: 10px;
-	min-height: 100%; // fill available height
-	max-height: 100%; // scroll inside
+	height: 100%;
```

Comment:
Please keep the min-height and max-height values, so it's less likely we break stuff. Adding `height: 100%;` should be ok.

Reply:
that's totally fine. I've reverted the deleted lines. :+1: